llvm/test/FileCheck/empty-variable-name.txt

```diff
@@ -0,0 +1,32 @@
+a
+
+; RUN: not FileCheck -color=0 -input-file %s %s 2>&1 | \
```

Comment:
Does the failing build bot set `FILECHECK_OPTS=--color`? The idiom is to set `%ProtectFileCheckOutput`.

I believe `-color=0` is not needed.

Reply:
Yes, see `FILECHECK_OPTS=-dump-input-filter=all -vv -color` here: https://lab.llvm.org/buildbot/#/builders/139/builds/60549/steps/2/logs/stdio

It's also clear that colors are enabled from the failing output.

IMHO even if the bot shouldn't set this and if it should be changed, the `-color=0` is still nice to have since if a test relies on assumption that colors are disabled - we'd better have the option explicitly disabled.

Reply:
I disagree. There are many `CHECK.*error:` tests and they don't use `-color=0` at all.

Reply:
OK, it makes sense. Deleted `-color=0` and used `%ProtectFileCheckOutput` instead. See 3bbb7c11f3c1e15cf7430fe543813c277649999b